alembic/script.py.mako

```diff
@@ -16,6 +17,19 @@ from alembic import op
 import sqlalchemy as sa
 ${imports if imports else ""}
 
+
+try:
+    is_sqlite = op.get_context().dialect.name == 'sqlite'
+except:
+    is_sqlite = False
```

Comment:
Although we use sqlite for our unit tests, is there any situation where we'd ever run migrations on a SQLite database?

No real objection to this code being here I suppose, mostly just curious.